service/src/main/kotlin/fi/espoo/evaka/backupcare/BackupCareQueries.kt

```diff
@@ -44,12 +44,12 @@ SELECT
   group_id,
   daycare_group.name AS group_name,
   daterange(backup_care.start_date, backup_care.end_date, '[]') AS period,
-  days_in_range(daterange(backup_care.start_date, backup_care.end_date, '[]')) - days_with_service_need AS missingServiceNeedDays
+  days_in_range(daterange(greatest(backup_care.start_date, '2020-03-01'::date), backup_care.end_date, '[]')) - days_with_service_need AS missingServiceNeedDays
```

Comment:
Adjusting only the start date with `greatest` can lead to invalid ranges and SQL exceptions if the end is before `2020-03-01`.

One way of doing this safely is using range intersection (= intersecting with a "validity date range"):

```
daterange(backup_care.start_date, backup_care.end_date, '[]') * daterange('2020-03-01', NULL)
```